portmidi.go

```diff
@@ -111,10 +86,7 @@ func Time() Timestamp {
 	return Timestamp(C.Pt_Time())
 }
 
+// convertToError converts a portmidi error code to a Go error.
 func convertToError(code C.PmError) error {
-	err, ok := errorMap[int(code)]
-	if !ok && code != 0 {
-		return ErrUnknown
-	}
-	return err
+	return errors.New(C.GoString(C.Pm_GetErrorText(code)))
```

Comment:
Are we leaking here? Once we convert the C string to Go string, should we deallocate?